server/service.go

```diff
@@ -162,6 +163,14 @@ func CAKeyPassword(pw []byte) ServiceOption {
 	}
 }
 
+// optional argument to set the validity of signed client certs in days
```

Comment:
the capass flag was necessary because without any password on the ca.key file the enroll process fails... do you know what is the default value in the sign function? maybe "" or sth. strange? or do you know what is the default ca.key password when using scepserver ca -init?